pkg/log/log_file_test.go

```diff
@@ -190,8 +197,8 @@ func TestFlags(t *testing.T) {
 		testFlags.Parse(test.args)
 		InitLogFileLimits(testFlags)
 		assert.Equal(t, test.maxSize, klog.MaxSize, test.name)
-		assert.Equal(t, test.maxNum, logFileMaxNum, test.name)
-		assert.Equal(t, test.logDir, logDir, test.name)
+		//assert.Equal(t, test.maxNum, logFileMaxNum, test.name)
+		//assert.Equal(t, test.logDir, logDir, test.name)
```

Comment:
? is this broken? something missing from the PR still?

Reply:
They are now added back.